app/policies/recipe_policy.rb

```diff
@@ -7,4 +7,8 @@ def update?
   def edit?
     update?
   end
+  
+  def fork?
+    user.present?
```

Comment:
Well, the creator of the recipe should not be able to fork his own recipes, right?

Reply:
Yes he can fork his own recipes, e.g. User posted his grandma original recipe, then forks to show his own modern version of it.